ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -209,7 +208,12 @@ export default class AlertTableRow extends React.Component {
       >
         <td className="table-width-xs px-1">
           <FormGroup check className="ml-2 pl-4">
+            <Label for={alert.id} hidden>
+              alert {alert.id} title
+            </Label>
             <Input
+              id={alert.id}
+              aria-label={`alert ${alert.id} title`}
```

Comment:
I don't think we need to have both an `aria-label` and an `aria-labelledby` (this can be removed since there isn't an element with an id that corresponds to the value being used).

Reply:
I'm wondering if the `for={alert.id}` will be read by screen readers along with the aria-label. Do we need both?